### ISSUE TITLE
[Remoto] Senior Full-Stack Software Engineer na HomeVision

### ISSUE THREAD

**Issue by mitchhomevision** [Gommunity/vagas#115]
## Descrição da vaga

Você trabalhará com um stack moderno de React/Typescript e Golang e desenvolverá produtos de automação de fluxo de trabalho para nossa base principal de usuários, composta por profissionais de avaliação imobiliária.

Você será encorajado a trabalhar na aplicação completa e não se restringirá a uma parte específica do stack. HomeVision engineers trabalham com produtos completos – o que inclui desde a produção de um modelo de avaliação até o desenvolvimento de visualizações de dados de front-end que ajudam a comunicar resultados de forma humana e intuitiva aos nossos usuários e clientes.

Essa posição exige alto nível de empatia com o usuário e capacidade de ter uma visão holística do produto. Você também deverá se sentir confortável em ajudar na orientação da equipe e liderar pelo exemplo, auxiliando na parceria com os cofundadores para assegurar processos eficientes e execução de alto nível.

## Local

Remoto

## Benefícios

- Salário competitivo e consistente
- Emprego a tempo inteiro
- A chance de aprender com uma equipe excepcional e internacional
- A oportunidade de moldar o futuro de nosso produto e negócios

## Requisitos

**Obrigatórios:**
- Ampla experiência em desenvolvimento full-stack de sistemas e frameworks back-end e front-end
- Experiência de trabalho comprovada em ambientes Unix/Linux e sistemas distribuídos
- Capacidade de aprender e dominar novas linguagens de programação e tecnologias
- Proficiência da língua inglesa em comunicação oral e escrita

**Desejáveis:**
- Experiência em uma ou mais tecnologias a seguir: Go, Python, React/Typescript, Elasticsearch e Docker
- Experiência em computação científica e numérica (por exemplo, numpy, scipy, scikit-learn, tensorflow, pytorch, jupyter) será considerada um diferencial

## Nossa empresa

Ajudamos a indústria de avaliação imobiliária a trabalhar com maior eficiência na análise de uma classe de ativos estimada em US$17 trilhões, trazendo equidade e transparência ao mercado imobiliário americano. Somos financiados pela [Initialized Capital](https://initialized.com/) e temos observado boa tração e aumento de receita. Nossa equipe está crescendo e procuramos um talentoso Full Stack Engineer. HomeVision é uma equipe apaixonada de engenheiros, cientistas de dados, designers e profissionais do setor imobiliário que reimagina o software de avaliação de imóveis por meio de ciência de dados e machine learning.

## Como se candidatar

Por favor envie um email para devjobs@homevision.co com seu CV anexado e "Senior Full-Stack Software Engineer" no título do e-mail

## Tempo médio de feedbacks

Costumamos enviar feedbacks em até 7-10 dias após cada processo.

## Labels

- Full-stack
- Remoto
- Sênior
- US

**Comment by delley** [comment]
Caso a vaga ainda esteja aberta, por favor, abrir uma nova issue!
